packages/material-ui/src/Slider/Slider.test.js

```diff
@@ -0,0 +1,470 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { spy, stub } from 'sinon';
+import { expect } from 'chai';
+import { createMount, getClasses } from '@material-ui/core/test-utils';
+import describeConformance from '@material-ui/core/test-utils/describeConformance';
+import { MuiThemeProvider, createMuiTheme } from '@material-ui/core/styles';
+import { cleanup, createClientRender, fireEvent } from 'test/utils/createClientRender';
+import consoleErrorMock from 'test/utils/consoleErrorMock';
+import Slider from './Slider';
+
+function touchList(touchArray) {
+  touchArray.item = idx => touchArray[idx];
+  return touchArray;
+}
+
+function fireBodyMouseEvent(name, properties = {}) {
```

Comment:
What is the difference to `fireEvent.click(document.body, properties)`?

Reply:
I have tried fireEvent first, it doesn't work. I haven't looked at why.

Reply:
Ok, I have found why it didn't work, JSDom doesn't implement the full TouchList API, notably .item(x) is not supported. I have changed the implement, removing `fireBodyMouseEvent`.